lib/writer.js

```diff
@@ -1,6 +1,10 @@
 var es = require('event-stream');
 var util = require('util');
 var extend = require('lodash.assign');
+var fs = require('fs');
+var normalizeData = require('normalize-package-data');
+var parseGitHubURL = require("github-url-from-git");
+var url_ = require("url");
```

Comment:
would it be horrible to alphabetize this list? and why `url_` instead of just `url`?